.github/workflows/ci.yml

```diff
@@ -269,6 +275,12 @@ jobs:
         . $GITHUB_WORKSPACE/.github/scripts/install_cargo_pgx.sh
         install_cargo_pgx
 
+    - name: Install llvm-tools-preview
+      run: rustup component add llvm-tools-preview
```

Comment:
```suggestion
      run: rustup component add llvm-tools-preview rustc-dev
```

Reply:
Happy to add that, but it doesn't look like it's required for CI...

can we have `plrustc/build.sh` just do this if rustup is available?

Reply:
CI didn't need this, but I'll merge in your suggested changes anyways.